README.md

```diff
@@ -85,56 +85,59 @@ To quickly try out RedisTimeSeries, launch an instance using docker:
 docker run -p 6379:6379 -it --rm redis/redis-stack-server:latest
 ```
 
-### Build and Run it yourself
+### Build it yourself
 
 You can also build and run RedisTimeSeries on your own machine.
 
 Major Linux distributions as well as macOS are supported.
 
-#### Requirements
-
-First, clone the RedisTimeSeries repository from git:
+You should have Redis installed, of course. The following, for example, builds Redis on a clean Unubtu docker image (`docker pull ubuntu`):
 
 ```
-git clone --recursive https://github.com/RedisTimeSeries/RedisTimeSeries.git
+mkdir ~/Redis
+cd ~/Redis
+apt-get update -y && apt-get upgrade -y
+apt-get install -y wget make pkg-config build-essential
+wget https://download.redis.io/redis-stable.tar.gz
+tar -xzvf redis-stable.tar.gz
+cd redis-stable
+make distclean
+make
+make install
```

Comment:
We should keep the `git clone --recursive https://github.com/RedisTimeSeries/RedisTimeSeries.git`